local-plugins/README.md

```diff
@@ -0,0 +1,27 @@
+# Nx Local Plugin Guide: tRPC
```

Comment:
should we also name the folder `local-plugins-trpc` or something, just to disambiguate? I feel we could easily have another `local-plugin` example in the future :)

Reply:
done!